extended/src/main/java/apoc/ml/bedrock/Bedrock.java

```diff
@@ -0,0 +1,123 @@
+package apoc.ml.bedrock;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import apoc.Description;
+import apoc.result.ObjectResult;
+import apoc.util.ExtendedUtil;
+import apoc.util.Util;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import static apoc.ml.bedrock.BedrockInvokeConfig.MODEL_ID;
+import static apoc.util.JsonUtil.OBJECT_MAPPER;
+import static apoc.ml.bedrock.BedrockInvokeResult.*;
+import static apoc.ml.bedrock.BedrockUtil.*;
+
+
+public class Bedrock {
+    
+    @Procedure
```

Comment:
let's put the explicit procedure name into the `@Procedure` annotation, better and safer for refactorings.